BrowserKit/Sources/Common/Theming/DefaultThemeManager.swift

```diff
@@ -83,132 +78,102 @@ public final class DefaultThemeManager: ThemeManager, Notifiable {
                                        UIApplication.didBecomeActiveNotification])
     }
 
-    // MARK: - ThemeManager
-
-    public func windowNonspecificTheme() -> Theme {
-        switch getNormalSavedTheme() {
-        case .dark, .nightMode, .privateMode: return DarkTheme()
-        case .light: return LightTheme()
-        }
-    }
-
-    public func windowDidClose(uuid: WindowUUID) {
-        windows.removeValue(forKey: uuid)
-        windowThemeState.removeValue(forKey: uuid)
-    }
-
-    public func setWindow(_ window: UIWindow, for uuid: WindowUUID) {
-        windows[uuid] = window
-        updateSavedTheme(to: getNormalSavedTheme())
-        updateCurrentTheme(to: fetchSavedThemeType(for: uuid), for: uuid)
-    }
-
-    public func currentTheme(for window: WindowUUID?) -> Theme {
+    // MARK: - Themeing general functions
+    public func getcurrentTheme(for window: WindowUUID?) -> Theme {
```

Comment:
Minor camel case typo

```suggestion
    public func getCurrentTheme(for window: WindowUUID?) -> Theme {
```